src/IB/IIMethod.cpp

```diff
@@ -4378,6 +4435,9 @@ IIMethod::getFromInput(Pointer<Database> db, bool /*is_from_restart*/)
         d_default_interp_spec.use_nodal_quadrature = db->getBool("interp_use_nodal_quadrature");
     else if (db->isBool("IB_use_nodal_quadrature"))
         d_default_interp_spec.use_nodal_quadrature = db->getBool("IB_use_nodal_quadrature");
+    if (db->isBool("use_current_mesh_configuration"))
```

Comment:
```suggestion
    if (db->isBool("use_current_mesh_configuration_for_interactions"))
```
Make sure this agrees with the database key name.